### PR TITLE
Add documentation for YesSQL breaking change

### LINKED ISSUE #12653: After upgrade  getting error : The certificate chain was issued by an authority that is not trusted
After recent upgrade with  YesSQL v 3.2.0 (with schema support) I am getting the following error when loading an existing project : 

image

The fix is to append to the connectionstring the setting : 
```
;TrustServerCertificate=True
```
but I am wondering if this could be related to yessql upgrade ?

### LINKED ISSUE #12679: Setup page not work, No error information is recorded when the database connection fails
First of all, I found that I could not add tenants in my program. I kept prompting that the connection string was incorrect or the server was unavailable. In fact, there was no problem with the database server.
My project uses OC 1.5.0-preview-17106

And Setup page not work.
After filling in all the contents, submit without prompting any error, and still stay on the Setup  page

image

To verify this problem, I pulled the latest code and ran the installation program directly using the latest source code. The same problem applies

### PR DESCRIPTION
**Pull request by Skrypt**
Fixes #12653, Fixes #12679

### CODE REVIEW

**Review by MikeAlhayek — APPROVED** [review]

**Comment by Skrypt** [comment]
Weird licence/cla limitation.

**Review by hishamco — CHANGES_REQUESTED** [review]

File: src/docs/releases/1.5.0.md [hunk 1]
```diff
@@ -58,4 +58,12 @@ ManageIndexes will be now ManageLuceneIndexes
...
\ No newline at end of file
+| Stored | Stored | Nothing to do. Should work as before |
+
+## YesSQL breaking changes
+
+YesSql is now using the new SQL client which encrypts the connection by default.
+
+If the SQL Server does not have a trusted SSL certificate and you would and want to continue using encrypted communicated using the untrusted server, add `TrustServerCertificate=True` to the connection string.
+
+If you do not want to not encrypt the communication at all, you can use `Encrypt=false`.
```
[the remark this answers is not in the document]
Comment (Skrypt) [reply src/docs/releases/1.5.0.md]: Fixed and also changed on the issue comment where I took that sentence from.

**Comment by Skrypt** [comment]
Anyone else than me can merge this?

**Review by hishamco — APPROVED** [review]

**Comment by ns8482e** [comment]
> Weird licence/cla limitation.

It seems something broken(may be auth) between dnf cla service and orchard cms org

**Comment by Skrypt** [comment]
I just tested this here with Elsa and it works:
https://github.com/elsa-workflows/elsa-core/pull/3379

So, it is something broken with our CI or our Github config.

**Comment by jtkech** [comment]
@Skrypt 

About the cla maybe related to github settings of the repo, would need to have access rights.

After reading some related issues, try to merge dev in your branch and push it to see if it unlocks this cla check, maybe it needs to be in sync with the last dev whose last commits are not marked as verified.

Not sure at all, just an idea ;)

**Comment by ns8482e** [comment]
This status you get for previously successful pr  https://cla.dotnetfoundation.org/OrchardCMS/OrchardCore?pullRequest=12671

**Comment by jtkech** [comment]
Oops

**Comment by Skrypt** [comment]
hmm ok, this is not good. How can the license from DNF disappear like this?

**Pull request merged**